src/backend/aspen/workflows/import_gisaid_isls/save.py

```diff
@@ -36,7 +37,11 @@ def save():
                 GisaidMetadata.gisaid_epi_isl,
             )
             .select_from(Sample)
-            .join(GisaidMetadata, Sample.public_identifier == GisaidMetadata.strain)
+            .join(
+                GisaidMetadata,
+                func.regexp_replace(Sample.public_identifier, "^hcov-19/", "", "i")
```

Comment:
the `^` here means "at the beginning of the string" and the `i` argument means "case insensitive"